cmd/ocsp-updater/main.go

```diff
@@ -395,25 +406,51 @@ func (updater *OCSPUpdater) missingReceiptsTick(batchSize int) {
 			continue
 		}
 	}
-
+	return nil
 }
 
 type looper struct {
-	clk       clock.Clock
-	stats     statsd.Statter
-	batchSize int
-	tickDur   time.Duration
-	tickFunc  func(int)
-	name      string
+	clk                  clock.Clock
+	stats                statsd.Statter
+	batchSize            int
+	tickDur              time.Duration
+	tickFunc             func(int) error
+	name                 string
+	recentFailures       int64
+	failureBackoffFactor float64
+	failureBackoffMax    time.Duration
+}
+
+func (l *looper) calculateBackoff() time.Duration {
```

Comment:
Please pull in [`core.RetryBackoff`](https://github.com/letsencrypt/boulder/pull/987/files#diff-4e55d08679303945433550ff2d2cebccR539) rather than reimplementing it here.

Note that RetryBackoff doesn't maintain state the same way this does. I think the nicer thing would be to just maintain a local failure counter in `loop()`'s for loop, and use that as input to `RetryBackoff`.